DOCS-2004: Add notice that cloud SLAM is temporarily disabled to cartographer (#2697)
viamrobotics/docs: docs/mobility/slam/cartographer/_index.md

Creating and updating SLAM maps with Cartographer is especially CPU-intensive, so the `cartographer` modular resource runs in the cloud for these two tasks.
For doing pure localization on an existing map, the `cartographer` modular resource runs locally on your machine.

{{% alert title="Info" color="tip" %}}

> Running `cartographer` in the cloud is temporarily disabled.

Author: @kkufieta Let me know if you want me to add any more information here. Should I instruct them to set the config_param "use_cloud_slam": `false` in our instructions or is that handled?

Reviewer: Maybe something along those lines:

```suggestion
Running `cartographer` in the cloud is temporarily disabled, and the attribute `use_cloud_slam` defaults to `false`. If `use_cloud_slam` is set to true, cloud slam will run in the background, but it won't be possible to see the results in the remote control card.
```

@EshaMaharishi let me know if this is ok.

Creating and updating SLAM maps with Cartographer is especially CPU-intensive, so the `cartographer` modular resource runs in the cloud for these two tasks.
For doing pure localization on an existing map, the `cartographer` modular resource runs locally on your machine.

{{% alert title="Info" color="tip" %}}

Running `cartographer` in the cloud is temporarily disabled, and the attribute `use_cloud_slam` defaults to `false`.

> If you set `use_cloud_slam` to true, cloud SLAM will run in the cloud, but it won't be possible to see the results in the remote control card.

Author: ```suggestion
If you set `use_cloud_slam` to true, it will not be possible to see the results in the remote control card.
```
